From 0290cd997f07b9c9e99ad2a7356dbe1f71488cf0 Mon Sep 17 00:00:00 2001
From: Jaroslav Kysela <jkysela@redhat.com>
Date: Wed, 3 Sep 2014 07:30:15 -0400
Subject: [alsa] hda/realtek: Add support of ALC288 codec

Message-id: <140972823573-266-patchset-jkysela@redhat.com>
Patchwork-id: 90966
O-Subject: [RHEL 7.1 PATCH 265/442] ALSA: hda/realtek - Add support of ALC288 codec
Bugzilla: 1112200
RH-Acked-by: John Linville <linville@redhat.com>
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Ivan Vecera <ivecera@redhat.com>

Signed-off-by: Kailang Yang <kailang@realtek.com>
Cc: <stable@vger.kernel.org>
Signed-off-by: Takashi Iwai <tiwai@suse.de>

(picked from kernel: 7c66593286bcd153e4868383e675673a27071bd5)
rhbz#1112200
Signed-off-by: Jaroslav Kysela <jkysela@redhat.com>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/sound/pci/hda/patch_realtek.c b/sound/pci/hda/patch_realtek.c
index 66e5195..7df6775 100644
--- a/sound/pci/hda/patch_realtek.c
+++ b/sound/pci/hda/patch_realtek.c
@@ -4911,6 +4911,7 @@ static int patch_alc269(struct hda_codec *codec)
   spec->codec_variant = ALC269_TYPE_ALC285;
   break;
  case 0x10ec0286:
+ case 0x10ec0288:
   spec->codec_variant = ALC269_TYPE_ALC286;
   break;
  case 0x10ec0255:
@@ -5780,6 +5781,7 @@ static const struct hda_codec_preset snd_hda_preset_realtek[] = {
  { .id = 0x10ec0284, .name = "ALC284", .patch = patch_alc269 },
  { .id = 0x10ec0285, .name = "ALC285", .patch = patch_alc269 },
  { .id = 0x10ec0286, .name = "ALC286", .patch = patch_alc269 },
+ { .id = 0x10ec0288, .name = "ALC288", .patch = patch_alc269 },
  { .id = 0x10ec0290, .name = "ALC290", .patch = patch_alc269 },
  { .id = 0x10ec0292, .name = "ALC292", .patch = patch_alc269 },
  { .id = 0x10ec0293, .name = "ALC293", .patch = patch_alc269 },
-- 
1.7.1